app/assets/main/react_components/footprint_form/FootprintForm.jsx

```diff
@@ -153,31 +161,15 @@ const FootprintForm = ({ calculator, questions, options, footprint }) => {
     else {
       do{
         questionIndex--;
-      } while(calculator[(order[questionIndex]).concat("_options")] !== undefined 
-      && !calculator[(order[questionIndex]).concat("_options")]);    
-    }
-  }
-
-  /**
-   * Called when the answer to a question is given, saves the result and loads the next question
-   */
-  function onAnswerGiven(givenAnswer){
-    saveAnswer(givenAnswer);
-    increaseIndex();
-    if(questionIndex == -1){
-      submit();
-    } else {
-      if (order[questionIndex-1] === "car_type" && givenAnswer === "no_car" ){
-        saveAnswer("");
-      }
-      setQuestion();
+      } while(calculator[(questionKeys[questionIndex]).concat("_options")] !== undefined 
+      && !calculator[(questionKeys[questionIndex]).concat("_options")]);    
     }
   }
 
   /**
    * Called on go back-button, loads the previous question by decreasing index and setting the question and options
    */
-  function onGoBack(){
+   function onGoBack(){
```

Comment:
Verkar ha smugit sig in ett mellanslag
```suggestion
  function onGoBack(){
```